qucs/qucs/octave_window.cpp

```diff
@@ -119,7 +113,30 @@ bool OctaveWindow::startOctave()
   // append process PATH, othewise Octave does not find gnuplot
   QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
   env.insert("PATH", env.value("PATH") + sep + QucsSettings.BinDir );
+#ifdef __MINGW32__
+  env.insert("PATH", env.value("PATH") + sep + QucsSettings.OctaveBinDir.canonicalPath() );
```

Comment:
> I haven't found other way than `PATH`

Using the `PATH` environment variable is perfectly fine and correct. you do not need to change qucs for that. `QucsSettings.OctaveBinDir` does not make any sense, because that's what `PATH` is for. PATH is explained everywhere on the internet, first hit [here](http://superuser.com/questions/284342/what-are-path-and-other-environment-variables-and-how-can-i-set-or-use-them).

you likely fell for a misconception in line 115. bad ideas are viral, don't repeat that mistake...